README.md

```diff
@@ -23,7 +23,7 @@ Dependencies in Fedora/RedHat:
 
 Dependencies in Ubuntu/Debian:
 
-    sudo apt-get install cmake libglib2.0-0 libglib2.0-dev libigraph1 libigraph-dev
+    sudo apt-get install cmake libglib2.0-0 libglib2.0-dev libigraph-dev
```

Comment:
By similar logic, would we also want to remove `libglib2.0-0`?

More broadly, I don't understand the standard conventions here enough to know if we should list all of the dependencies even when the binaries are pulled in by the header packages, or not. Do you know if there is such a guideline published somewhere?

Reply:
Yes, I think so: the -dev package includes the proper dependencies already.

```
$ apt-cache  depends libglib2.0-dev
libglib2.0-dev
  Depends: libffi-dev
  Depends: libglib2.0-0
  Depends: libglib2.0-bin
  Depends: libglib2.0-dev-bin
  Depends: libmount-dev
  Depends: libpcre3-dev
  Depends: libselinux1-dev
  Depends: pkg-config
    pkgconf
  Depends: zlib1g-dev
  Breaks: <libglib2.0-0-dbg>
  Suggests: libgirepository1.0-dev
  Suggests: libglib2.0-doc
  Replaces: <libglib2.0-0-dbg>
```

My general sense is that -dev packages include their binary packages as dependencies, although I can't seem to find documentation stating this is a firm rule when building Debian/Ubuntu packages.

(Including `libigraph1` was an issue because on some version of Ubuntu, that version of libigraph wasn't available, and libigraph-dev instead depended on a different version.)